source/geometry/FloatBox2D.ooc

```diff
@@ -41,7 +41,18 @@ FloatBox2D: cover {
 	bottomCenter ::= FloatPoint2D new(this center x, this bottom)
 	hasZeroArea ::= this size hasZeroArea
 	area ::= this size area
-	init: func@ (=leftTop, =size)
+	init: func@ (.leftTop, .size) {
+		if (size x < 0) {
+			leftTop x += size x
+			size x = -size x
+		}
+		if (size y < 0) {
+			leftTop y += size y
+			size y = -size y
+		}
+		this leftTop = leftTop
+		this size = size
```

Comment:
According to https://github.com/cogneco/ooc-kean/issues/814 we want to avoid that local variables are called like class variables. I guess that is why we usually do not use the `.` for the input variables of the functions.